src/Appwrite/Stats/Usage.php

```diff
@@ -232,12 +269,12 @@ private function createOrUpdateMetric(string $projectId, int $time, string $peri
             $document = $this->database->getDocument('stats', $id);
             if ($document->isEmpty()) {
                 $this->database->createDocument('stats', new Document([
-                    '$id' => $id,
-                    'period' => $period,
-                    'time' => $time,
-                    'metric' => $metric,
-                    'value' => $value,
-                    'type' => $type,
+                '$id' => $id,
+                'period' => $period,
+                'time' => $time,
+                'metric' => $metric,
+                'value' => $value,
+                'type' => $type,
```

Comment:
Should this be indented?

Reply:
@lohanidamodar, should this be indented?

Reply:
Should be resolved.